statistics/cmsketch.go

```diff
@@ -180,6 +181,21 @@ func (c *CMSketch) findTopNMeta(h1, h2 uint64, d []byte) *TopNMeta {
 	return nil
 }
 
+// MemoryUsage returns the total memory usage of a CMSketch in B.
+// We ignore the size of other metadata in CMSketch
+func (c *CMSketch) MemoryUsage() (sum int64) {
+	if c == nil {
```

Comment:
If `c == nil`, `nil pointer panic` will happen when invoking c.MemoryUsage.
Thus we do not need to check it here.